iceoryx_utils/test/moduletests/test_posix_timer.cpp

```diff
@@ -25,6 +25,7 @@
 
 using namespace ::testing;
 
+using namespace iox::units;
```

Comment:
Please do not start `using namespace`. Will lead to namespace pollution very easily, when used in other places

Reply:
@mossmaurice This cannot happen in cpp files. The `using namespace` statement is only a problem in header files since everyone who is including this header file is also executing the `using namespace` statement.

A cpp file is on the other hand a compilation unit where no one knows how the implementation looks like since no one is including a cpp file. You can access symbols in a cpp file by knowing that they are present (done in the header) and linking against them later. 

But `using namespace` is not a symbol. It is a compile time directive to access a certain scope so we do not have a problem there!

Reply:
@elfenpiff is correct, it is fine this way.